packages/tools/devtools/devtools-core/src/ContainerDevtools.ts

```diff
@@ -281,15 +300,15 @@ export class ContainerDevtools implements IContainerDevtools, HasContainerKey {
 		[ConnectContainer.MessageType]: async (untypedMessage) => {
 			const message = untypedMessage as ConnectContainer.Message;
 			if (message.data.containerKey === this.containerKey) {
-				this.container.connect();
+				this.container.connect?.();
```

Comment:
We'll need to make these behaviors more robust before merging. We should update things such that the UI doesn't display the connect/disconnect buttons when they are not available.